main/cgi.c

```diff
@@ -34,7 +34,7 @@ CgiStatus ICACHE_FLASH_ATTR cgiLed(HttpdConnData *connData) {
 	len=httpdFindArg(connData->post.buff, "led", buff, sizeof(buff));
 	if (len!=0) {
 		currLedState=atoi(buff);
-		ioLed(currLedState);
+		gpio_set_level(LED_CGI, currLedState);
```

Comment:
should this be using an ioLed() call instead, as you've introduced those abstractions for the led control?

Reply:
I know it is inconsistent.  I didn't like the abstraction that was there as it seemed unnecessary to introduce another variable to index the LEDs.
I'll adjust to make it consistent by letting ioLed take a GPIO number as input.